composer.json

```diff
@@ -65,7 +65,6 @@
         "drupal/fieldblock": "^2.0@alpha",
         "drupal/file_browser": "^1.3",
         "drupal/flood_control": "^2.2",
-        "drupal/gtranslate": "^1.14",
```

Comment:
I'd imagine the deployment will fail trying to uninstall a module that it can't find in the vendor folder. You probably want to leave `drupal/gtranslate` here but kill the patch. Then later, after the module has been disabled, you can remove the composer dependency.

Reply:
So this is what was causing the autoload error I presume. I wasn't gathering any useful error messages from circleCI that I could find.